templates/_schemas/details/default.twig

```diff
@@ -2,14 +2,29 @@
 
 
 
+{% set graph = create('Spatie\\SchemaOrg\\Graph') %}
+{% set seomateMetaData = craft.seomate.getMeta({
+  config: { includeSitenameInTitle: false },
+}|merge(seomate ?? {}, true)) %}
 {% set entryImages = entryImages ?? craft.assets.relatedTo(entry).kind('image').all() %}
```

Comment:
This variable is not used anywhere. Remove?

Reply:
Updated the `image` parameter in `defaultSchemaProps` and now the variable is used.